.helm-staging.yml

```diff
@@ -17,3 +17,10 @@ nodeSelector:
 providers:
   github:
     comment_footer: "_If you have feedback about this comment, please, [tell us](%s)._"
+
+analyzers:
+  - name: style-analyzer
+    addr: ipv4://lookout-style-analyzer:2000
+
+repositories:
```

Comment:
This setting is not going to have any effect as this key is not taken into account. What are you trying to achieve here?

Reply:
Thank you for notice! That is a way of adding more repositories for lookout to watch ([example](https://github.com/src-d/lookout/blob/master/config.yml.tpl))

Reply:
Well, the chart does not currently support that... I will add it to the chart.

Reply:
Actually for some reason the repositories are defined with the positional argument, and this section of the config file is only used to optionally provide the credentials for those repos.
[Looking at the chart](https://github.com/src-d/charts/blob/master/lookout/templates/lookout-deployment.yaml#L38) I believe should be done with `repository` (already up in this file)

```
app:
  lookout:
    repository: github.com/src-d/lookout,github.com/bblfsh/uast-viewer
```